broker/common/types.go

```diff
@@ -32,16 +33,21 @@ type SchemaManager interface {
 // QueryExecutor defines query executor
 type QueryExecutor interface {
 	// Execute executes query and flush result to connection
-	Execute(namespace, sqlQuery string, w http.ResponseWriter) (err error)
+	Execute(ctx context.Context, namespace, sqlQuery string, w http.ResponseWriter) (err error)
 }
 
 // BlockingPlanNode defines query plan nodes that waits for children to finish
 type BlockingPlanNode interface {
-	Run() (queryCom.AQLQueryResult, error)
+	Run(ctx context.Context) (queryCom.AQLQueryResult, error)
```

Comment:
maybe all change to use Execute() for consistency?

Reply:
changed